🚑(dependency) pin django-treebeard to 4.4 (#1284)
openfun/richie: CHANGELOG.md

- Fix autosuggest to redirect user directly on the course page when a course
  entry has been selected from the suggestion dropdown

> - As django-treebeard 4.5 introduces major issues, we decide to pin it to 4.4
>   waiting to identify clearly what was wrong and where comes from this issue

Reviewer: ```suggestion
- Pin `django-treebeard` to `4.4` as the 4.5 release introduces BC
```
